appspec-scripts/stop_app.sh

```diff
@@ -1,9 +1,35 @@
 #!/bin/bash
```

Comment:
can we put this link as a comment here as a reminder of the codedeploy hook sequence? https://docs.aws.amazon.com/codedeploy/latest/userguide/reference-appspec-file-structure-hooks.html#appspec-hooks-server

haha mainly for future reference to remind me and the team that stop_app always run first